Unit_Conversion_Calculator.py

```diff
@@ -3,10 +3,12 @@
 Date: 12/7/2023
 Author: Liam Cassell
 All Rights Reserved 
-Revision #: 0
+Revision #: 1 (Updated program as to provide campatiblity for testing, fixed imperial volume calculations)
 """
 
-def userInput(): #Function to get the users input numeric amount, input unit type, desired unit output
+test = 'This is a test for PR'
+
+def userInput(): #Function to get the users input numeric amount, input unit type, desired unit output.
     A = metricUnit
     B = imperialUnit

```

Comment:
Have a look at this post on how to name your variables: https://realpython.com/python-variables/#variable-names

If you use A and B down further down in the codebase, you'd probably forget what they are, so give a meaningful variable name.